apps/src/templates/studioHomepages/CoteacherInviteNotification.jsx

```diff
@@ -0,0 +1,116 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+import i18n from '@cdo/locale';
+import Notification, {NotificationType} from '@cdo/apps/templates/Notification';
+import {asyncLoadCoteacherInvite} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+import Button from '../Button';
+import {BodyTwoText, StrongText} from '@cdo/apps/componentLibrary/typography';
+import HttpClient from '@cdo/apps/util/HttpClient';
+import DCDO from '@cdo/apps/dcdo';
+
+export const showCoteacherInviteNotification = coteacherInvite => {
+  return !!coteacherInvite && DCDO.get('show-coteacher-ui', false);
```

Comment:
That shouldn't effect whether this will show up early since it defaults to false and the main purpose of the flag is to merge things gradually into develop. If we need to manually modify the flags with a deploy later when we want to turn it on, I think that would be fine.

Reply:
Still in that state; specifically, DCDO flag changes only get picked up by the web server when it gets restarted, which in our build generally only happens as part of a deploy. Sounds like it this case it's okay for a change to this flag to take on the order of hours rather than seconds to resolve, so you should be good!


I do have a [proposed fix](https://github.com/code-dot-org/code-dot-org/pull/53833) out for review which will get things working again, but we're not yet entirely sure about the performance implications. Hopefully DCDO should be back to 100% functionality soon!